src/Router.js

```diff
@@ -7,13 +7,16 @@ import CustomNavBar from './shared/CustomNavBar/CustomNavBar';
 import LogIn from './components/LogIn/LogIn';
 import Dashboard from './components/Dashboard/Dashboard'
 import AlertsNotification from './components/AlertsNotifications/AlertNotifications';
+import GlobalMap from './components/GlobalMap/GlobalMap'
+import ShipDetailsMap from './components/GlobalMap/ShipDetailsMap/ShipDetailsMap'
```

Comment:
I would rename everything that contains word Ship to Vessel because stena web is referring to as vessel everywhere